systemtest/errorgrouping_test.go

```diff
@@ -48,7 +48,9 @@ func TestErrorGroupingName(t *testing.T) {
 
 	var names []string
 	for _, hit := range result.Hits.Hits {
-		names = append(names, gjson.GetBytes(hit.RawSource, "error.grouping_name").String())
+		values := hit.Fields["error.grouping_name"]
```

Comment:
Note for reviewers: I made this change because in the Elasticserach plugin, we are using a script field which is evaluated at index time. It ends up in `fields`, but not in `_source`.